charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -175,4 +175,4 @@ spec:
       - "ReadWriteOnce"
       resources:
         requests:
-          storage: 10Gi
+          storage: 20Gi
```

Comment:
That change will break all existing clusters - it is not allowed by default to change the volume size. Can you please again elaborate why you need this? If it's really necessary then we should think about a way to do it only for alicloud.

Reply:
I will fix it in a better way. The reason is that 20Gi is the minimum size of Disk that Alicloud can provision.